codalab/worker/worker_run_state.py

```diff
@@ -251,13 +252,26 @@ def mount_dependency(dependency, shared_file_system):
 
         dependencies_ready = True
         status_messages = []
+        dependency_keys_to_paths: Dict[DependencyKey, str] = dict()
```

Comment:
rm dependency_keys_to_paths, we don't use it

Reply:
We still use it.